--- a/config/baseconfig.mk	2018-01-12 19:51:52.076706568 -0400
+++ b/config/baseconfig.mk.new	2018-01-13 17:08:54.465360852 -0400
@@ -4,7 +4,7 @@
 # whether a normal build is happening or whether the check is running.
 includedir := $(includedir)/$(MOZ_APP_NAME)-$(MOZ_APP_VERSION)
 idldir = $(datadir)/idl/$(MOZ_APP_NAME)-$(MOZ_APP_VERSION)
-installdir = $(libdir)/$(MOZ_APP_NAME)
+installdir = $(libdir)/$(MOZ_APP_NAME)-beta
 sdkdir = $(libdir)/$(MOZ_APP_NAME)-devel-$(MOZ_APP_VERSION)
 ifeq (.,$(DEPTH))
 DIST = dist
